### ISSUE TITLE
Unexpected reserved word  when using with Meteor 1.3

### ISSUE THREAD

**Issue by kokokenada** [pbastowski/meteor-typescript#5]
Not sure if I missed a step (in which case some instructions would be helpful) or if there is an bug.

In any event, when I try to use this with 1.3, I get this error during the build:

``` meteor
[[[[[ ~/apps/fastcards ]]]]]                  

=> Started proxy.                             
=> Started MongoDB.                           
W20160409-11:34:44.671(-4)? (STDERR)          
W20160409-11:34:44.672(-4)? (STDERR) packages/common.js:23
W20160409-11:34:44.672(-4)? (STDERR) export const name = 'common';
W20160409-11:34:44.672(-4)? (STDERR) ^^^^^^
W20160409-11:34:44.673(-4)? (STDERR) SyntaxError: Unexpected reserved word
W20160409-11:34:44.673(-4)? (STDERR)     at /Users/kenono/apps/fastcards/.meteor/local/build/programs/server/boot.js:278:30
W20160409-11:34:44.673(-4)? (STDERR)     at Array.forEach (native)
W20160409-11:34:44.673(-4)? (STDERR)
```

Here is the repo: https://github.com/kokokenada/tstest2

**Comment by pbastowski** [comment]
You need to either:
1. rename `common.js` to `common.ts` and then TypeScript will compile the code.
or
2. add `api.use('pbastowski:angular-babel');` to your package.js. 

Adding `pbastowski:angular-babel` to your project does not automatically make it available to any packages you create, which is why you get the error.

**Comment by kokokenada** [comment]
Thanks for the reply.  Adding 

```
api.use('pbastowski:angular-babel'); 
```

to package.js did indeed work and I can now build and run. AWESOME!

The next bump I hit (again, I probably missed a configuration step), is that the compiler and the Atom editor both miss an error.  

```
let testArray = ['a', 'b'];
let index = 1
console.log('testArray[index]:' + testArray[indexx]);  // should be index, not indexx
```

The problem is that error is not flagged until runtime.

I added the fix and a new file ‘client/test.ts’ which has a variable spelled wrong to the repository.

**Comment by kokokenada** [comment]
I also noticed when I rename client/main.js to main.ts (in https://github.com/kokokenada/tstest2), that the import can no longer resolve angular.  Is there a different way to locate angular in a ts file comapred to a js file?  I tried installing typings etc. (more less following this approach http://www.namekdev.net/2015/11/beginning-with-meteor-and-angularjs-on-top-of-typescript/), but was unable to resolve.

**Comment by pbastowski** [comment]
No, there isn't. `import ... from` works the same in Babel and TypeScript. Typings don't work with my TypeScript package, because it doesn't read anything from the tsconfig.json file, the project configuration file, except for what's documented in the README on github/AtmosphereJs.

I believe `meteortypescript:compiler` provides a fuller TypeScript implementation, if you need such things as typings.

**Comment by kokokenada** [comment]
Hmm. Odd that just renaming a .js to .ts would cause imports to break...

**Comment by pbastowski** [comment]
I cloned your repo.
npm install
And it runs fine for me with either `main.js` or `main.ts`.

Also, change your code in main.js/.ts to this:

``` js
import 'angular';
import 'angular-meteor';

angular.module('fastcards', [
  'angular-meteor',
  'common'
]);
```

**Comment by kokokenada** [comment]
I followed the same procedure and can confirm that that did indeed fix that problems.  THANK YOU!

I'm still curious as to why the build doesn't complain about the deliberate errors in test.ts.  Visual Studio Code sees them as errors. (Atom doesn't but I presume that's a config problem on my end.)

I've committed your corrections and left the deliberate errors in case anyone is interested.

Thanks again.
